### PR TITLE
Add LWJGL

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/376520/
https://steamdb.info/app/99900/

### Brief explanation of the change
Detect lwjgl.jar

### CODE REVIEW

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.LWJGL**

These apps now match: **(75)** [93200](https://steamdb.info/app/93200/depots/) [99900](https://steamdb.info/app/99900/depots/) [99920](https://steamdb.info/app/99920/depots/) [108600](https://steamdb.info/app/108600/depots/) [111800](https://steamdb.info/app/111800/depots/) [200570](https://steamdb.info/app/200570/depots/) [203210](https://steamdb.info/app/203210/depots/) [207710](https://steamdb.info/app/207710/depots/) [217710](https://steamdb.info/app/217710/depots/) [219190](https://steamdb.info/app/219190/depots/) [219200](https://steamdb.info/app/219200/depots/) [221020](https://steamdb.info/app/221020/depots/) [221030](https://steamdb.info/app/221030/depots/) [244770](https://steamdb.info/app/244770/depots/) [255320](https://steamdb.info/app/255320/depots/) [259620](https://steamdb.info/app/259620/depots/) [263360](https://steamdb.info/app/263360/depots/) [264910](https://steamdb.info/app/264910/depots/) [286340](https://steamdb.info/app/286340/depots/) [303340](https://steamdb.info/app/303340/depots/) [318080](https://steamdb.info/app/318080/depots/) [319430](https://steamdb.info/app/319430/depots/) [321600](https://steamdb.info/app/321600/depots/) [324080](https://steamdb.info/app/324080/depots/) [328080](https://steamdb.info/app/328080/depots/) [329240](https://steamdb.info/app/329240/depots/) [330100](https://steamdb.info/app/330100/depots/) [335180](https://steamdb.info/app/335180/depots/) [336300](https://steamdb.info/app/336300/depots/) [339010](https://steamdb.info/app/339010/depots/) [340880](https://steamdb.info/app/340880/depots/) [340980](https://steamdb.info/app/340980/depots/) [342560](https://steamdb.info/app/342560/depots/) [344000](https://steamdb.info/app/344000/depots/) [346770](https://steamdb.info/app/346770/depots/) [363180](https://steamdb.info/app/363180/depots/) [363460](https://steamdb.info/app/363460/depots/) [364240](https://steamdb.info/app/364240/depots/) [366590](https://steamdb.info/app/366590/depots/) [366630](https://steamdb.info/app/366630/depots/) [368970](https://steamdb.info/app/368970/depots/) [377290](https://steamdb.info/app/377290/depots/) [380870](https://steamdb.info/app/380870/depots/) [392610](https://steamdb.info/app/392610/depots/) [414510](https://steamdb.info/app/414510/depots/) [416790](https://steamdb.info/app/416790/depots/) [439960](https://steamdb.info/app/439960/depots/) [442710](https://steamdb.info/app/442710/depots/) [444160](https://steamdb.info/app/444160/depots/) [459760](https://steamdb.info/app/459760/depots/) [517110](https://steamdb.info/app/517110/depots/) [548650](https://steamdb.info/app/548650/depots/) [583490](https://steamdb.info/app/583490/depots/) [583700](https://steamdb.info/app/583700/depots/) [601400](https://steamdb.info/app/601400/depots/) [647960](https://steamdb.info/app/647960/depots/) [656740](https://steamdb.info/app/656740/depots/) [748880](https://steamdb.info/app/748880/depots/) [796660](https://steamdb.info/app/796660/depots/) [805520](https://steamdb.info/app/805520/depots/) [822660](https://steamdb.info/app/822660/depots/) [999030](https://steamdb.info/app/999030/depots/) [1114870](https://steamdb.info/app/1114870/depots/) [1235160](https://steamdb.info/app/1235160/depots/) [1268290](https://steamdb.info/app/1268290/depots/) [1290480](https://steamdb.info/app/1290480/depots/) [1333810](https://steamdb.info/app/1333810/depots/) [1336430](https://steamdb.info/app/1336430/depots/) [1560480](https://steamdb.info/app/1560480/depots/) [1719570](https://steamdb.info/app/1719570/depots/) [1772630](https://steamdb.info/app/1772630/depots/) [2327230](https://steamdb.info/app/2327230/depots/) [2366190](https://steamdb.info/app/2366190/depots/) [2367280](https://steamdb.info/app/2367280/depots/) [2490750](https://steamdb.info/app/2490750/depots/)

Please confirm that all these changed apps are correct.

File: tests/types/Engine.LWJGL.txt
Comment (Lifeismana) [inline tests/types/Engine.LWJGL.txt:1]: add some real strings

**Review by Lifeismana — CHANGES_REQUESTED** [review]
add some non matching tests

**Comment by Lifeismana** [comment]
merged in 605399fcd689b7530b25f515409499cb77c01891

**Pull request closed without merging**